spark/v2.4/build.gradle

```diff
@@ -162,7 +163,6 @@ project(':iceberg-spark:iceberg-spark-runtime') {
     relocate 'org.apache.orc', 'org.apache.iceberg.shaded.org.apache.orc'
     relocate 'io.airlift', 'org.apache.iceberg.shaded.io.airlift'
     // relocate Arrow and related deps to shade Iceberg specific version
-    relocate 'io.netty.buffer', 'org.apache.iceberg.shaded.io.netty.buffer'
```

Comment:
The Spark dependency on netty hasn't changed since Spark 2.4?

Reply:
Looks like there was a 3.9.9.Final dependency in 2.4.0.

Reply:
2.4.0 bundles `io.netty:netty-all:4.1.17.Final`, see https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.11/2.4.0

Reply:
netty3 and netty4 classes use different package name, they can co-exist.